src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -334,7 +338,10 @@ private static double[] ComputeArrayDeltas(double[] a, double[] b)
 
         #endregion
     }
+}
 
+namespace Microsoft.ML.Data
```

Comment:
In general I like having one file per type, but am fine when we group a bunch of related types into one file.

However, having types from multiple namespaces in one file feels like it should really get split out into a separate file.

Reply:
That's fine. I think what I'll do is I'll have two commits in my next push so it remains easy to review what i have done, one where I make changes, and another where I split out the file.

Reply:
OK @eerhardt this is now done. Note that the commit "Move metrics statistics types to new file." is just a cut and paste job, no internal code was changed.